src/apis/blockchain_api.ts

```diff
@@ -112,6 +112,21 @@ export class BlockchainAPI {
         });
     }
 
+    /**
+     * Eventually returns the first address from available accounts, which we assume to be the
```

Comment:
This doesn't make sense to me. `getCurrentUser` actually doesn't make sense. It's not a "user", it's a default account - maybe it should be `getCurrenAccount`

Reply:
Because one user can have multiple accounts.